src/pybind/mgr/dashboard/frontend/src/app/core/error/error.component.scss

```
@@ -0,0 +1,53 @@
+@use './src/styles/defaults/bootstrap-defaults' as bd;
+
+h4 {
+  color: bd.$gray-700;
+}
+
+i {
+  font-size: 6em;
+  margin-top: 120px;
+}
+
+.text-center {
+  background-color: bd.$body-bg-alt;
+}
+
+.dashboard {
+  background-color: bd.$body-bg-alt;
+  height: 100%;
+  position: relative;
+}
+
+.content {
+  left: 50%;
+  position: absolute;
+  top: 40%;
+  transform: translate(-50%, -50%);
+  width: 100%;
+}
+
+.row {
+  display: block;
+  margin-left: -29px;
+  margin-right: -29px;
+  padding-top: 10em;
+}
+
+.fa-exclamation-triangle {
+  color: bd.$danger;
+}
+
+.fa-lock {
+  color: bd.$gray-600;
+}
+
+.fa-wrench {
+  color: bd.$info;
+}
+
+::ng-deep .btn-light {
```

This will take care of the btn-light issue. Also its best not to use the !important tag on the child level scss components.

```suggestion
::ng-deep cd-back-button {
  .btn-light {
    background-color: bd.$info;
    color: bd.$white;
    overflow: visible;
    
    &:hover {
      background-color: bd.$info;
      color: bd.$black;
    }
  }
}
```